replica/database.cc

```diff
@@ -1013,16 +1013,13 @@ void database::remove(const table& cf) noexcept {
     }
 }
 
-future<> database::drop_column_family(table& cf, timestamp_func tsf, bool snapshot) {
+future<> database::drop_column_family(table& cf) {
```

Comment:
Maybe it's now better named detach_column_family(), since it just disassociates the table from the APIs, bit it keeps existing in the background.

Reply:
(and it's just a small part of the drop procedure)

Reply:
For bonus points return the foreign_ptr here. So this function detaches ownership from database and gives it to the caller. But that's just cosmetic, ignore if it causes problems.